app/forms/recovery_code_form.rb

```diff
@@ -25,9 +25,9 @@ def valid_recovery_code?
     recovery_code_generator.verify(code)
   end
 
-  def result
+  def extra_analytics_attributes
     {
-      success: success,
+      method: 'recovery code',
```

Comment:
Before, `method: 'recovery code'` was merged into the FormResponse result hash in the controller. The purpose of this string is to differentiate between the 3 different types of Multi-Factor Auth: OTP, TOTP, and Recovery Code. All MFA controllers write to the same `MULTI_FACTOR_AUTH` event. The `method` key allows us to differentiate between the different types, or methods, of MFA.

If we left this in the controller like it was, would its meaning be more clear? I'm struggling to come up with a constant name that makes sense. `RECOVERY_CODE_MFA_METHOD` that is set to the String "recovery code" feels weird to me.

If the issue is that the meaning is not clear, I think renaming the Hash key would be sufficient:

```ruby
{
  multi_factor_auth_method: 'recovery code'
}
```